napari_clusters_plotter/_plotter.py

```diff
@@ -146,7 +148,21 @@ class MyNavigationToolbar(NavigationToolbar):
     def __init__(self, canvas, parent):
         super().__init__(canvas, parent)
         self.canvas = canvas
-
+    
+    def _update_buttons_checked(self):
+        super()._update_buttons_checked()
+        # changes pan/zoom icons depending on state (checked or not)
+        if 'pan' in self._actions:
+            if self._actions['pan'].isChecked():
+                self._actions['pan'].setIcon(QIcon(os.path.join(os.getcwd(), "images", "my_toolbar_icons", "Pan_checked.png")))
```

Comment:
Hi Marcelo @zoccoler ,

the icons should live within the plugin directory (e.g. "napari_clusters_plotter\icons") so that they are deployed together with the code to the end-user. Btw. `os.getcwd()` may point to the folder from which napari was started. Consider using `__file__` like [here](https://github.com/clEsperanto/napari_pyclesperanto_assistant/blob/master/napari_pyclesperanto_assistant/_gui/_button_grid.py#L6) which points to the folder when the current python file lives. 🌻

Best,
Robert

Reply:
Hi Robert @haesleinhuepf ,

Thanks, I had used relative paths previously, but your suggestion of taking the path from `__file__` is indeed a more accurate solution. I have implemented that now.
About the icons, I had put them inside the plugin at 'images\my_toolbar_icons' before. Nevertheless, I have moved them to this new directory 'napari_clusters_plotter\icons' for simplicity and perhaps as being a more appropriate location.

Obs: left some prints, we should remove them before merging if this works.